tests/MongoDBDriverReadPreference_error-003.phpt

```diff
@@ -1,21 +1,21 @@
 --TEST--
-MongoDB\Driver\ReadPreference construction (invalid maxStalenessMS)
+MongoDB\Driver\ReadPreference construction (invalid maxStalenessSeconds)
 --FILE--
 <?php
 require_once __DIR__ . '/utils.inc';
 
 echo throws(function() {
-    new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_PRIMARY, null, ['maxStalenessMS' => 1000]);
+    new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_PRIMARY, null, ['maxStalenessSeconds' => 1000]);
 }, 'MongoDB\Driver\Exception\InvalidArgumentException'), "\n\n";
 
 echo throws(function() {
-    new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_SECONDARY, null, ['maxStalenessMS' => -1]);
+    new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_SECONDARY, null, ['maxStalenessSeconds' => -1]);
```

Comment:
This will end up being a valid test case, since the spec says we should allow specifying the default value. In https://github.com/mongodb/mongo-php-driver/pull/458, I had changed this test case to use 0.